app/src/main/res/layout/activity_main.xml

```diff
@@ -63,7 +92,49 @@
                     android:background="@color/white"
                     android:divider="@null"
                     android:dividerHeight="0dp"
-                    android:elevation="@dimen/elevation_card"/>
+                    android:elevation="@dimen/elevation_card">
+                </ListView>
+
+                <android.support.constraint.ConstraintLayout
+                    android:id="@+id/empty_message_view"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:visibility="gone">
+
+                    <ImageView
+                        android:id="@+id/imageView"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        app:layout_constraintBottom_toBottomOf="parent"
+                        app:layout_constraintEnd_toEndOf="parent"
+                        app:layout_constraintStart_toStartOf="parent"
+                        app:layout_constraintTop_toTopOf="parent"
+                        app:layout_constraintVertical_bias="0.326"
+                        app:srcCompat="@mipmap/no_messages" />
+
+                    <TextView
+                        android:id="@+id/textView"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="@dimen/meshim_margin_small"
+                        android:text="@string/empty_message_text1"
+                        android:textStyle="bold"
+                        app:layout_constraintEnd_toEndOf="parent"
+                        app:layout_constraintStart_toStartOf="parent"
+                        app:layout_constraintTop_toBottomOf="@+id/imageView" />
+
+                    <TextView
+                        android:id="@+id/textView8"
```

Comment:
could make the id tags more descriptive for these views (id should describe the purpose of the field)